specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3007,14 +3070,15 @@
           "type": "string"
         },
         "properties": {
+          "description": "Properties of a shared private link resource.",
           "$ref": "#/definitions/SharedPrivateLinkResourceProperty",
           "x-ms-client-flatten": true
         }
       }
     },
     "SharedPrivateLinkResourceProperty": {
```

Comment:
>SharedPrivateLinkResourceProperty [](http://example.com/codeflow?start=5&length=33)

this looks a lot like something we should ideally refactor in common-types later also, what do you think?

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2688199

Reply:
agreed, but I need more time to confirm changing to common-type won't break existing SDK